### PR TITLE
Feat: Auth

### PR DESCRIPTION
**Pull request by manojncsoni007**
Description:
- Added login & signup page
- Added login & signup functionality

Screenshot:
image
image

### CODE REVIEW

**Review by srishti-maurya — APPROVED** [review]
Looks good to me

File: src/pages/Auth/Auth.css [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+    gap: 0.3rem;
+}
+
+.input-container input{
+    /* width: 20rem; */
+    padding: 0.5rem;
+}
+
+.user-auth {
+    /* margin: 1.5rem; */
```
Comment (arpit01923) [inline src/pages/Auth/Auth.css:64]: remove unnecessary code

**Review by arpit01923 — APPROVED** [review]
- clean code
- good folder structure
- good to merge

File: src/pages/Auth/Auth.css (hunk 1 above)
Comment (manojncsoni007) [reply src/pages/Auth/Auth.css:64]: Ok

**Pull request merged**